### DISCUSSION TITLE
Run different <script> in header for each language

### DISCUSSION THREAD

**Discussion by SuzikoS** [qtranslate/qtranslate-xt#1349] [Q&A]
Hello i need run diffent script for each language in my website, but i cnat find how make it?

for example:

for EN language
<script type="text/javascript" >  script EN version  </script>
for FR language
<script type="text/javascript" >  script FR version  </script>

**Comment by Komarovski** [comment]
You can use this function `qtranxf_getLanguage()` :
```
<?php
if(qtranxf_getLanguage() == 'uk'){
   echo 'Привіт';
}elseif(qtranxf_getLanguage() == 'ru'){
   echo 'Привет';
}elseif(qtranxf_getLanguage() == 'en'){
   echo 'Hello';
}else{
   echo 'Bonjour';
}
?>
```

**Reply by SuzikoS** [reply]
Thank you
